spec.html

```diff
@@ -26620,6 +26620,28 @@ <h1>Abstract Module Records</h1>
             </tr>
           </table>
         </emu-table>
+
+        <emu-clause id="sec-EvaluateModuleSync" type="abstract operation">
+          <h1>
+            EvaluateModuleSync (
+              _module_: a Module Record,
+            ): either a normal completion containing ~unused~ or a throw completion
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It synchronously evaluates _module_, provided that the caller guarantees that _module_'s evaluation will return an already settled promise.</dd>
+          </dl>
+
+          <emu-alg>
+            1. Assert: _module_ is not a Cyclic Module Record.
+            1. Let _promise_ be ! _module_.Evaluate().
+            1. Assert: _promise_.[[PromiseState]] is either ~fulfilled~ or ~rejected~.
+            1. If _promise_.[[PromiseState]] is ~rejected~, then
+              1. Perform HostPromiseRejectionTracker(_promise_, "handle").
+              1. Return ThrowCompletion(_promise_.[[PromiseResult]]).
```

Comment:
```suggestion
              1. Throw _promise_.[[PromiseResult]].
```

more idiomatic

Reply:
I'll revert this change, as looking through other cases of `Throw` it's indeed never used to throw something that was stored in an existing variable.